docs/i18n/portuguese/how-to-help-with-video-challenges.md

```diff
@@ -76,11 +76,9 @@ Pick a challenge markdown file from the options above.
 First, find the videoId.
 
 For example, in the following code from the header of a video challenge markdown file, the videoId is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.
-```
+````
 ---
-id: 5e9a093a74c4063ca6f7c14d
-title: Data Analysis Example A
-challengeType: 11
+id: 5e9a093a74c4063ca6f7c14d title: Data Analysis Example A challengeType: 11
```

Comment:
What's up with this line?

Reply:
Crowdin seems to be doing something strange here (I think because the entire text is wrapped in four backticks and crowdin does not like that)